test/verify/check-machines-disks

```diff
@@ -651,6 +668,49 @@ class TestMachinesDisks(VirtualMachinesCase):
         if m.image in ["debian-testing"]:
             self.allow_journal_messages('.* type=1400 .* apparmor="DENIED" operation="open" profile="libvirt.* name="%s.*' % self.vm_tmpdir)
 
+    def testAddDiskCustomPath(self):
+        b = self.browser
+        m = self.machine
+
+        self.createVm("subVmTest1")
+
+        # Prepare file for Custom File disk type
+        m.execute("touch /var/lib/libvirt/novell.iso")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual machines")
+
+        b.wait_in_text("#vm-subVmTest1-state", "Running")
+        self.goToVmPage("subVmTest1")
+
+        if "debian" not in m.image and "ubuntu" not in m.image:
```

Comment:
Why debian and ubuntu can't attach cdroms?

Reply:
I think there was a problem year ago with apparmor not allowing custom file paths for disks. But I don't really remember it anymore. I run works well. So I deleted the condition